assets/python/ml/pyml:setup_variables_packages.pyi

```diff
@@ -158,3 +158,26 @@ class Context(object):
 context = Context()
 
 is_using_train_test_split = "is_using_train_test_split" in context and (context.load("is_using_train_test_split"))
+
+# Create a Class for a DummyScaler()
+class DummyScaler():
+    """
+    This class is a 'DummyScaler' which trivially acts on data by returning it unchanged.
+    """
+
+    def __init__(self):
+        pass
+
+    def fit(self, X):
+        return self
+
+    def transform(self, X):
+        return X
+
+    def fit_transform(self, X):
+        return X
+
+    def inverse_transform(self, X):
+        return X
+
+context.save(DummyScaler(), 'target_scaler')
```

Comment:
We import this file at the start of every unit, which means this overwrites the `target_scaler` at the start of every unit. Not desired behavior if a user already has a scaler in their workflow.

The easy fix here is just adding a check to only save it if `target_scaler` doesn't already exist.